projects/cashmere/src/lib/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,6 +1,6 @@
 <ol class="breadcrumb">
     <li *ngFor="let breadcrumb of _breadcrumbs; let last = last" class="breadcrumb-item">
-        <a *ngIf="!last" [routerLink]="[breadcrumb.url, breadcrumb.params]" [innerText]="breadcrumb.label"></a>
+        <a *ngIf="!last" [routerLink]="[breadcrumb.url" queryParamsHandling="preserve" [innerText]="breadcrumb.label"></a>
```

Comment:
I would really like to see this as an option in the component, users of the library may not want this behavior

Reply:
Good call @benjanderson !